MobiFlight/Joysticks/JoystickInput.cs

```diff
@@ -16,6 +18,7 @@ public class JoystickInput
         /// <summary>
         /// The input's type. Supported values: Button, Axis.
         /// </summary>
+        [JsonConverter(typeof(StringEnumConverter))]
```

Comment:
This needs to be added so when re-exporting the object to JSON for validation after migration the enum is written out as a string (as the schema expects) and not a number.